docusaurus/docs/packages/pkg/relayer/_category_.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Did you run `docusarus` locally to make sure it looks good?

Reply:
Yes